Sunday August 25th 2013 12:07 pm -04:00,Although it works fine on my system but the only reason that this error can pop up is if you are using ord both as a built in function and as an identifier. See the example below:>>> ord(c)99>>> ord=9>>> ord(c)Traceback (most recent call last):  File <pyshell#9> line 1 in <module>    ord(c)TypeError: int object is not callableTry it again after restarting the shell., Resolved,0
Sunday August 25th 2013 3:24 pm -04:00,Strings are usually enclosed in quotes. Check the Python documentations as it says They can be enclosed in single quotes (...) or double quotes (...) with the same resulthttp://docs.python.org/3/tutorial/introduction.html#strings, Resolved,0
Monday August 26th 2013 11:03 am -04:00,HiIts not about function ord. Actually Strings are usually enclosed in quotes. Request you to check the Python tutorial in the below mentioned link wherein it says that String can be enclosed in single quotes (...) or double quotes (...) with the same result.http://docs.python.org/3/tutorial/introduction.html#strings, Resolved,1
Wednesday September 11th 2013 1:11 pm -04:00,I also first tried  return 12 <temp <28  and it works just as fine. Why do we have to use or when mathematical functions are suppose to be faster?>>> 12 < 14 < 28True>>> 12 < 10 < 28False>>> 12 < 30 < 28False, Unresolved,0
